src/mint/NUT12.ts

```diff
@@ -0,0 +1,20 @@
+import { hash_e, createRandomPrivateKey, DLEQ } from '../common/index.js';
+import { ProjPointType } from '@noble/curves/abstract/weierstrass.js';
+import { bytesToHex, numberToBytesBE } from '@noble/curves/abstract/utils';
+import { secp256k1 } from '@noble/curves/secp256k1';
+import { bytesToNumber, hexToNumber } from '../util/utils.js';
+
+export function createDLEQProof(B_: ProjPointType<bigint>, a: Uint8Array): DLEQ {
```

Comment:
```suggestion
/**
 * !!! WARNING !!! Not recommended for production use, due to non-constant time operations
 * See: https://github.com/cashubtc/cashu-crypto-ts/pull/2 for more details
 */
export function createDLEQProof(B_: ProjPointType<bigint>, a: Uint8Array): DLEQ {
```

Let's do something like this for now..
@lollerfirst could you add some resources/links to the PR that give some more insight into the problem with these operations? 

Other than that LGTM

Reply:
@gandlafbtc could put there the link to the wikipedia page on timing attacks?